modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/admin/js/components/export-form/ExportFormModal.tsx

```diff
@@ -32,36 +32,41 @@ const ExportFormModal: React.FC<IProps> = ({
 				{Liferay.Language.get('export')}
 			</ClayModal.Header>
 
-			<form action={exportFormURL} method="post">
-				<ClayModal.Body>
+			<ClayModal.Body>
+				<form 
+					action={exportFormURL} 
+					id="exportFormEntriesFormId" 
```

Comment:
I am no sure about this string. I have never seen a ID string ending in "Id", this is already implicit. You don't add "Id" suffixes to ID strings. On the bright side, down at the `<button>` `form` attribute, having "Id" kind of makes things clearer for beginners programmers who may not know that this is referencing a form ID. I would say: remove the suffix, make it only "exportFormEntriesForm". What do you think?